frontend/pulse3d/src/components/basicWidgets/UsageProgressWidget.js

```diff
@@ -123,17 +129,19 @@ export default function UsageProgressWidget({ colorOfTextLabel }) {
         <ProgressDiv>
           <p>Usage</p>
           <CircularProgressWithLabel value={usagePercentage} colorOfTextLabel={colorOfTextLabel} />
-          <ProgressP>{`${actualAnalyses ? actualAnalyses : 0}/${maxAnalyses} Analysis used`}</ProgressP>
+          <ProgressLabel>{`${
+            actualAnalyses ? actualAnalyses : 0
```

Comment:
nitpick, could do `actualAnalyses || 0`

Reply:
I didn't notice that, was briefly changing the html tags